api/v1/report.proto

```diff
@@ -114,33 +114,32 @@ message LogEntry {
     EMERGENCY = 800;
   }
 
+  // The name of the LogEntryDescriptor providing the schema for this log entry.
+  string descriptor_name = 1;
```

Comment:
checking: are these names fully qualified?  in other words, is this expected to be "service.example.io/logEntries/default" or just "default" (and proxy/mixer will be able to find the right descriptor)?